src/main.spec.js

```diff
@@ -36,21 +36,64 @@ test("Checks HTMLElements", () => {
       name: "FinalDir",
       absolute_path: "/mnt/project/FinalDir",
       type: "directory",
-      children: null
+      children: [
+        {
+          name: "yarn.lock",
+          absolute_path: "/mnt/project/vendor/yarn.lock",
+          type: "file",
+          children: null
+        },
+        {
+          name: "package.json",
+          absolute_path: "/mnt/project/vendor/package.json",
+          type: "file",
+          children: null
+        }
+      ]
     }
   ];
   const container = document.createElement("div");
   fsapi.render(input, container);
 
-  const ul = container.querySelector("ul");
+  const trees = container.querySelectorAll("ul");
 
-  for (const child of ul.children) {
+  for (const child of trees[0].children) {
     expect(child).toBeInstanceOf(HTMLLIElement);
   }
-  expect(ul.children[0].textContent).toBe("Dir");
-  expect(ul.children[1].textContent).toBe("FinalDir");
-  expect(ul.children[2].textContent).toBe("AnotherFile.html");
-  expect(ul.children[3].textContent).toBe("SFile.sth");
-  expect(ul.children[4].textContent).toBe("test.go");
-  expect(ul.children[5].textContent).toBe("Troll.go");
+  expect(trees[0].children[0].children[0].textContent).toBe("Dir");
```

Comment:
Can we rename `trees[0]` to something more semantic (e.g. `rootTree`, `mainTree`)